client/client-multi/client-c/src/kaa/kaa_logging.c

```diff
@@ -463,9 +527,29 @@ kaa_error_t kaa_logging_handle_server_sync(kaa_log_collector_t *self
 
         remove_request(self, bucket_id);
 
+        /* Count logs left to upload */
+
+        uint32_t total_logs = 0;
+        kaa_list_for_each(kaa_list_begin(self->timeouts),
+                          kaa_list_back(self->timeouts),
+                          sum_log_records, &total_logs);
+
+        kaa_log_bucket_info_t log_bucket_info = {
+            .log_count = total_logs,
```

Comment:
There should number of delivered logs. This information you can get on remove_request(self, bucket_id) stage by changing a little bit this function.